src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -78,4 +78,7 @@ func (client *Client) Configure(cfg *ClientConfig) {
 	client.MinQOS = cfg.MinQOS
 	bal := client.initBalancer(cfg)
 	client.initReverseProxy(bal, cfg.DumpHeaders)
+	client.priorCfg = cfg
+	client.priorTrustedCAs = &x509.CertPool{}
+	*client.priorTrustedCAs = *globals.TrustedCAs
```

Comment:
@atavism This is something that was broken from before your changes.  I noticed that we never stored the old client configuration, so everytime any part of the system configuration changed, the client would reconfigure itself.  This fixes that.